discord/ext/bridge/core.py

```diff
@@ -591,6 +591,8 @@ async def convert(self, ctx, arg: bool):
 
 
 class BridgeOption(Option, Converter):
+    """Slash command option for bridge commands. Otherwise equivalent in every way to :class:`discord.Option`."""
```

Comment:
> Slash command option for bridge commands

That makes it sound like it doesn't apply to prefixed command counterparts. How about something like this?

```suggestion
    """Represents a selectable slash command option and a prefixed command argument for bridge commands.
    """
```